docs/references/database/list-documents.md

```diff
@@ -1 +1 @@
-Get a list of all the user documents. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of the project's documents. [Learn more about different API modes](/docs/admin).
\ No newline at end of file
+Get a list of all the user's documents in a given collection. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of documents belonging to the provided collectionId. [Learn more about different API modes](/docs/admin).
```

Comment:
> @[lohanidamodar](https://github.com/lohanidamodar) lohanidamodar [16 hours ago](https://github.com/appwrite/appwrite/pull/3391#discussion_r903168202)
as i think now this is misleading. even on admin mode, the endpoint returns list of all of the documents in the collection and database provided, not the project right?


> @[eldadfux](https://github.com/eldadfux) eldadfux [12 hours ago](https://github.com/appwrite/appwrite/pull/3391#discussion_r903275462)
Maybe: Get a list of all the user's documents in the given database.